r/R/anime.R

```diff
@@ -13,7 +13,34 @@ validate_lines <- function(x, error_call = rlang::caller_call()) {
   geoarrow::as_geoarrow_array(x)
 }
 
+# TODO: use arrow and nanoarrow imports in a proper function
+demo_use_of_arror_and_nanoarrow <- function() {
```

Comment:
Please delete this function as it isn't needed

Reply:
Any ideas how to make the checks happy without it?

Reply:
With the function:

```
── R CMD check results ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── anime 0.0.0.9000 ────
Duration: 30.1s

❯ checking installed package size ... NOTE
    installed size is 23.7Mb
    sub-directories of 1Mb or more:
      libs  23.5Mb

0 errors ✔ | 0 warnings ✔ | 1 note ✖
```

Reply:
After removing the (I agree pointless, other than to make `check()` happy) function with https://github.com/JosiahParry/anime/pull/7/commits/d70935645c065f38b645f7dd5543ffdd529ef664:

Reply:
```
❯ checking dependencies in R code ... NOTE
  Namespaces in Imports field not imported from:
    ‘arrow’ ‘nanoarrow’
    All declared Imports should be used.
```

Note, not a warning, can live with that. Plan is to use them in future, or are they used in ways I'm not seeing?

Reply:
They're being used on the rust side.